crates/shared/src/order_validation.rs

```diff
@@ -490,6 +495,9 @@ impl OrderValidating for OrderValidator {
 pub struct SignatureConfiguration {
     pub eip1271: bool,
     pub eip1271_skip_creation_validation: bool,
+    /// Additional flat gas fee to be applied if signature is not EIP-1217
+    /// or if EIP-1271 validation is skipped on creation.
+    pub flat_gas_fee: U256,
```

Comment:
I think this is a good idea in general. I would not include it in this PR though.

Specifically, I think we should add this (since it is a separate change, and EIP-712, Eth-sign, EIP-1271 and pre-sign all have slightly different gas characteristics) but:
- The different overheads are deterministic - so we should measure them instead of making them configurable
- We should also differentiate between EIP-712, Eth-sign, and pre-sign
- Not strictly needed here, since `verification_gas_limit` is the gas of the `isValidSignature` call, and not related to the different gas overheads associated to each signing scheme.

Reply:
Ok, great, I will remove these changes. You left a TODO for configuring the default value when it's not EIP-1271 signature, that's why I've added it here.

Reply:
Removed.